src/ClientData/CallstackData.cpp

```diff
@@ -151,7 +170,10 @@ void CallstackData::UpdateCallstackTypeBasedOnMajorityStart() {
       const auto& frames = callstack.frames();
       ORBIT_CHECK(!frames.empty());
       uint64_t outer_frame = *frames.rbegin();
-      ++count_by_outer_frame[outer_frame];
+      if (!IsPcInFunctionsToStopAt(absolute_address_to_size_of_functions_to_stop_unwinding_at,
+                                   outer_frame)) {
+        ++count_by_outer_frame[outer_frame];
```

Comment:
Shouldn't `++count_for_this_thread` also be conditioned by this? It's used below with `kFilterSupermajorityThreshold`. Let's say that all the complete callstacks, 26% are in `__wine_syscall_dispatcher`, 70% in the thread start function, and 4% are actually wrong. We still want to filter the 4%.

Reply:
That's actually a good catch. I was already wondering why I needed to add so many events to the test. Now that actually works with fewer. Done.